### ISSUE TITLE
Dropout: A Simple Way to Prevent Neural Networks from Overfitting

### ISSUE THREAD

**Issue by sooooner** [sooooner/papers#17]
# Dropout: A Simple Way to Prevent Neural Networks from Overfitting[[link](https://jmlr.org/papers/v15/srivastava14a.html)]

## Introduction
DNN에서 overfitting을 해결하려는 여러 시도들이 있었지만 연산량을 고려하지 않았을때 가장좋은 방법은 여러 파라미터 조합의 결과를 가중 평균내는 것이다.(앙상블?)

연산량을 줄이면서 성능을 향상시키는 Dropout을 제안한다

##  Model Description
![image](https://user-images.githubusercontent.com/31437443/108801985-83d49700-75da-11eb-9079-6347d299d630.png)

layer에서 사용할 unit을 랜덤하게 뽑으면서 학습을 진행한다.

![image](https://user-images.githubusercontent.com/31437443/108802184-02c9cf80-75db-11eb-85c7-4dd3deff81b8.png)

여기서 r은 Bernoulli 확률 분포에서 뽑힌 값이다

### Backpropagation
각 batch에서 선택된 unit들만 학습
Max-norm regularization이 효율이 좋다고 한다. 이는gradient exploding 방지해주기 때문이라고 한다.

## Experimental Results
![image](https://user-images.githubusercontent.com/31437443/108802523-e1b5ae80-75db-11eb-9297-46e07d69589f.png)
효과 굳!

## Salient Features
실험결과를 통해 성능개선을 입증하였으니 drop out이 network에 미치는 영향을 조사함

생성 된 기능의 품질에 대한 드롭 아웃의 영향을 분석합니다. 
드롭 아웃이 은닉 유닛 활성화의 희소성에 어떤 영향을 미치는지 확인합니다.
또한 드롭 아웃에서 얻은 이점이 유닛 보유 확률, 네트워크 크기 및 훈련 세트의 크기에 따라 어떻게 다른지 확인합니다. 
이러한 관찰은 드롭 아웃이 왜 그렇게 잘 작동하는지에 대한 통찰력을 제공합니다.

###  Effect on Features
 standard neural network에서 backprop은 다른 unit들이 수행하는 것을 감안하여 loss를 낮추도록 훈련된다.
즉, 다른 unit의 실수를 수정하는 방식으로 변경될 수 있다. 이는 오버피팅으로 이어진다.

드롭 아웃은 서로 신뢰하지 못하게 하기 때문에 독립적인 기능을 하도록 도와줌

![image](https://user-images.githubusercontent.com/31437443/108803476-1c204b00-75de-11eb-9860-d2cebbb637bd.png)

 b를 보면 이미지의 다른 부분에서 edges, strokes and spots을 감지하는 것처럼 보입니다. 이는 drop out이 co-adaptations을 깨뜨리는 것을 보여줍니다. 이것이 아마도  generalization errors를 낮추는 주된 이유 이다.

## Effect on Sparsity
drop out의 부작용: 정규화 항이 없는경우 unit의 활성이 희박해짐 

## Conclusion
**Dropout is a technique for improving neural networks by reducing overfitting.**
